.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+Thank you for contributing to the `fabric-chaincode-evm` project!
+
+Please make sure you have done the following before submitting:
+
+* [ ] Ran the basic checks: `make basic-checks`
+
+* [ ] Ran the unit tests: `make unit-test`
```

Comment:
why are these all separated by newlines? Should be all next to each other. It's a waste of vertical space otherwise. 
First interaction with this is going to be in a monospace buffer view, not a formatted view.

Reply:
I forgot the bullets when I initially made it, so I did new lines to separate them. Ill remove them